### ISSUE TITLE
Server Null Pointer - Exception getting block type in world

### ISSUE THREAD

**Issue by LeiserGeist** [Beyond-Reality/BeyondRealityModPack#198]
A server, left running for a while, regularly crashes and produces this null pointer.

I've yet to investigate the location of the error, but the report seems to show it's something trying to catch a block on fire in the nether (dim-1).

Update: After investigation, the error location was just a little fire on some netherrack that happened to be chunk loaded by a nether pump (hence the errors when nobody's even online).

Still no idea *why* it caused this, but I've removed the block and I'll see if it crashes again within a day or two.

I do have a handful of mods installed not default to the pack, consisting of:
- Blood Magic
- ChickenChunks
- Custom NPCs
- Funky Locomotion

Crash Report:
http://pastebin.com/i5waLKA0

**Comment by LeiserGeist** [comment]
After regenerating the area that had the fire erroring out, the server still crashes after just one night of running. Same error.

**Comment by Dominance** [comment]
Going to need the FML server logs that correspond to this crash. As the crash report basically has 0 modded features included.

**Comment by LeiserGeist** [comment]
Didn't even think about that. Sorry, finding it now...

**Comment by Dominance** [comment]
I need the full logs. If it just crashed it would be labeled fml-3. Use http://pastebin.ubuntu.com/ to paste the full log as pastebin wont allow for large logs.

**Comment by LeiserGeist** [comment]
Of course. http://paste.ubuntu.com/10056886/
Is this the ~~droid~~ log you're looking for?

Could be AutoSaveWorld. Gonna try AromaBackup instead.

**Comment by Dominance** [comment]
Seems like its Doom Like dungeons failing to generate a structure all the way. try using this latest build http://minecraft.curseforge.com/mc-mods/74246-project-74246/files/2226302. It will be going into the next BR version but until then have at it. Its a server side only mod so it shouldn't conflict with the clients using a different version.
also your auto save is pretty jacked up it seems lol.

**Comment by LeiserGeist** [comment]
I replaced the wonky autosave bukkit/cauldron plugin with AromaBackup (unfortunately, client side too) and it hasn't crashed.
I'll update DLD next time I take the server down but I'm going to try to keep it running to see if it ends up crashing.

And yeah, I was going to change the backup time to be much longer but I kept procrastinating and eventually forgetting, lol
Aroma seems to be much better written anyway, it doesn't cause any lag or TPS drops. (TPS drops have been non-existant with this pack, I'm quite impressed)
